src/components/CategoryCard.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+const CategoryCard = (props) => {
+  return (
```

Comment:
You can destructure your props, for cleaner code

Reply:
Yes, will do that. thanks